smartmin/management/__init__.py

```diff
@@ -194,4 +168,3 @@ def check_all_permissions(sender, **kwargs):
 
 post_migrate.connect(check_all_permissions)
 post_migrate.connect(check_all_group_permissions)
```

Comment:
We actually changed that a while back so it only does stuff once for the last app